test/e2e/__snapshots__/ClientOptions.test.js.snap

```diff
@@ -3,6 +3,7 @@
 exports[`Client console.log clientLogLevel is silent 1`] = `
 Array [
   "Hey.",
+  "[HMR] Waiting for update signal from WDS...",
```

Comment:
This is an issue we should look at separately, clientLogLevel silent is not correctly setting the log level of HMR. We simply were not picking up on it before in this test because it was not inlining HMR before